app/views/fact_check_content_review.scala.html

```diff
@@ -48,7 +48,7 @@ <h2 class="app-task-list__section">
         @for((page, index) <- review.pages.zipWithIndex) {
           <li class="app-task-list__item">
             <span class="app-task-list__task-name">
-              <a href="#" aria-describedby="page-@index">@page.title</a>
+              <a href="@{routes.FactCheckPageReviewController.onPageLoad(review.ocelotId, page.title.drop(1)).toString}" aria-describedby="page-@index">@page.title</a>
```

Comment:
You could also use substring(1), but that would be resorting to Java!

Reply:
.... did someone mention Java ????